email_templates/html/hypervisor_vm.html.j2

```diff
@@ -0,0 +1,9 @@
+Dear Cloud User {{username}}, <br/><br/>
+
+Test email about servers on hv <br/><br/><br/>
```

Comment:
@jacob-ward @DavidFair - What should the email template be here? Worth getting this nailed down before merging this into Prod

Reply:
Added templates for hv fault and hv maintenance emails